tsconfig.base.json

```diff
@@ -4,7 +4,7 @@
     "declaration": true,
     "declarationMap": true,
     "esModuleInterop": true,
-    "lib": ["ES2022"],
+    "lib": ["ES2022", "ESNext.disposable"],
```

Comment:
Does this change anything for our users? Will they have to make any changes to their config files?

Reply:
No, they'll just have to use TS 5.2. Given that we're now already at TS 5.4, IMO it's okay.

Reply:
I tested it with the wallet sdk and tests passed.